src/hooks/bridge/Chains.ts

```diff
@@ -28,11 +31,44 @@ export function useLiFiSwapChains() {
   });
 }
 
+export function useSquidChains() {
+  return useQuery(['squidChains'], async () => {
+    const response = await fetch(`${SQUID_API}/v1/chains`);
```

Comment:
can we use axios here as here also `src/state/pbridge/hooks.ts` you are using axios

Reply:
I directly used 0xSquid SDK.